grype/vulnerability/log.go

```diff
@@ -0,0 +1,13 @@
+package vulnerability
+
+import "github.com/anchore/grype/internal/log"
+
+// LogDropped is a hook called when vulnerabilities are dropped from consideration in a vulnerability Provider,
+// this offers a convenient location to set a breakpoint
+func LogDropped(vulnerabilityID string, reason any, context ...any) {
```

Comment:
can this be an internal concern to v6 instead of a public API helper function in the core vulnerability package?